modules/orca/variables.tf

```diff
@@ -175,4 +175,9 @@ variable "staged_recovery_queue_message_retention_time_seconds" {
 variable "status_update_queue_message_retention_time_seconds" {
   type        = number
   description = "The number of seconds status_update_queue SQS retains a message in seconds. Maximum value is 14 days."
+}
+
+variable "api_vpc_endpoint_id" {
```

Comment:
May want to call this just `vpc_endpoint_id` to remove confusion that this is the API Gateway id when in reality it is the NGAP VPC endpoint ID.  See previous notes on description recommendations.

Reply:
changed to vpc_endpoint_id